### PR TITLE
Completed API Pair Project

### PR DESCRIPTION
**Pull request by nnijat**

### CODE REVIEW

**Comment by nnijat** [comment]
Completed API Pair Project is for **Week7** project
Completed Fetch API Unite Test is for **Week8** project

**Comment by nnijat** [comment]
_Completed API Pair Project_ is for **Week7** project
_Fixed conflict and finalized the project_ is for **Week8** project

File: app.js [hunk 1]
```diff
@@ -1,2 +1,105 @@
...
+rl.question("Enter your beer ID: ", function (beerID) {
+    getBeerNameByID(fetch, beerID);
+    rl.close();
+});
+
+// IDs you can user : c4f2KE/zTTWa2/zfP2fK/xwYSL2/
+function getBeerNameByID(beerID) {
+    const URL = BASE_LINK + ENDPOINT + KEY + PARAMETER + beerID;
+    return fetch(URL)
+        //.then(res => res.json())
```
Comment (anthonyjsilva) [inline app.js:28]: I recommend removing unused commented-out code when you go to make your PR

**Comment by anthonyjsilva** [comment]
@nnijat 
Very nice!

**Pull request merged**